core/src/test/scala/unit/kafka/utils/MockScheduler.scala

```diff
@@ -35,21 +35,21 @@ import org.apache.kafka.common.utils.Time
  * Incrementing the time to the exact next execution time of a task will result in that task executing (it as if execution itself takes no time).
  */
 class MockScheduler(val time: Time) extends Scheduler {
-  
+
```

Comment:
Let's not make so many whitespace changes to this file... it is distracting when trying to read the patch.